### PR TITLE
DownloadGraph: Ignore SemVer parsing issues when sorting

### LINKED ISSUE #3295: TypeError: Invalid major version
Sentry Issue: [CRATES-IO-FRONTEND-P2](https://sentry.io/organizations/rust-lang/issues/2212224874/?referrer=github_integration)

```
TypeError: Invalid major version
  at new c (addon-tree-output/~ember-css-modules/ext.js:2:1)
  at e.exports (addon-tree-output/~ember-css-modules/ext.js:2:1)
  at None (addon-tree-output/~ember-css-modules/ext.js:2:1)
  at Array.sort (<anonymous>)
  at default (addon-tree-output/~ember-css-modules/ext.js:2:1)
...
(19 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by plaets**
Fixes #3295

### CODE REVIEW

File: app/components/download-graph.js [hunk 1]
```diff
@@ -106,7 +106,14 @@ export default class DownloadGraph extends Component {
       }
     });
 
-    let versionsList = semverSort([...versions]);
+    let versionsList = [...versions];
+    try {
+      semverSort(versionsList);
```
Comment (Turbo87) [inline app/components/download-graph.js:111]: does this sort the array in-place?

Comment (plaets) [reply app/components/download-graph.js:111]: I think it does, unless I'm misunderstanding something
https://github.com/rust-lang/crates.io/blob/master/app/components/download-graph.js#L8
https://github.com/npm/node-semver/blob/master/functions/sort.js#L2
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort
> The sort() method sorts the elements of an array in place

Comment (plaets) [reply app/components/download-graph.js:111]: I also checked how the code behaves when I comment out semverSort and it seems to confirm that

Comment (Turbo87) [reply app/components/download-graph.js:111]: okay, great, thanks :)

**Review by Turbo87 — APPROVED** [review]

**Pull request merged**